src/olympia/api/fields.py

```diff
@@ -414,13 +414,13 @@ def _fetch_some_translations(self, field, langs):
         base_locale = to_language(settings.LANGUAGE_CODE)
         translations = {}
         if base_locale in langs:
-            # we get the base_locale for free - it's just the field text
-            translations[base_locale] = str(field)
-            langs = (lang for lang in langs if lang != base_locale)
+            with override(base_locale):
+                translations[base_locale] = str(field)
+                langs = (lang for lang in langs if lang != base_locale)
```

Comment:
tbh I don't know if the 'en-US' .po is inadvertently loaded anyway before the middleware selects the language so this may all be a moot point!

Reply:
Yeah, I verified in a shell, then in a naked management command, and finally inside a request after a fresh restart: just initializing django loads the `en-US` catalog (... twice in the shell/management commad, once as `en-US` and once as `en-us`!).

Reply:
^ re-highlighting my "If we don't go with that above alternate suggestion" paragraph above

Reply:
mmm, but the way it's handled is different. For the `base_locale`, we call `str()`, for the rest `gettext()`. I wonder if it's always safe to `gettext()` all the locales even the base one...

Reply:
Also, since the languages is a set... but not guaranteed to be... to ensure the base one is first, we'd have to do something like (variables renamed to clarity)
```python
        languages = set(languages)
        try:
            languages.remove(base_language)
        except KeyError:
            pass
        languages = [base_language] + list(languages)
```

Reply:
This should do it:
```python
     if base_locale in langs:
            langs = [base_locale, *(lang for lang in langs if lang != base_locale)]
```

I'm not sure about the str() vs. gettext() though - I assumed it's pretty much the same thing?

Reply:
Ah, I thought you wanted to remove the whole condition, I see it now.

The `str()` won't work when the string passed is an actual string (like something coming from the database directly). The `gettext()` can be different from the actual base string in the base language if somehow there is a translation in that language that differs from the base string. I suppose it's fine...

Reply:
> The gettext() can be different from the actual base string in the base language if somehow there is a translation in that language that differs from the base string. I suppose it's fine...

so you're saying `gettext(field)` sometimes _doesn't_ work?  (if field is already a `gettext_lazy` I assume).  That seems bad.

Reply:
Actually the condition itself should be removed - we _always_ want to ensure the base language is present in first position.

Reply:
32a9342fca4a63c3e70457344737365012ddd36b